### PR TITLE
feat(cf worker): add test with Module syntax

### PR DESCRIPTION
**Pull request by Jolg42**
Related https://github.com/prisma/prisma/issues/15958

### CODE REVIEW

File: dataproxy/cloudflare-workers-module/wrangler.base.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Docs https://developers.cloudflare.com/workers/wrangler/configuration/
+
+name = "e2e-cloudflare-workers-module"
+main = "./src/index.ts"
+compatibility_flags = []
+workers_dev = true
+compatibility_date = "2022-05-03"
```
Comment (millsp) [inline dataproxy/cloudflare-workers-module/wrangler.base.toml]: Should we bump this?

File: dataproxy/cloudflare-workers-module/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      at async start
+  (/Users/j42/Dev/ecosystem-tests/dataproxy/cloudflare-workers-module/node_modules/.pnpm/wrangler@3.0.1/node_modules/wrangler/wrangler-dist/cli.js:147836:34)
+  {
+    text: 'A request to the Cloudflare API
+  (/accounts/cd5f1cdfd37dfbcd1c847c40a01c3f7a/workers/scripts/e2e-cloudflare-workers-module/edge-preview)
+  failed.',
+    notes: [
+      {
+        text: 'Uncaught Error: InvalidDatasourceError: Datasource "db" references an environment
+  variable "CLOUDFLARE_DATA_PROXY_URL" that is not set\n' +
```
Comment (millsp) [inline dataproxy/cloudflare-workers-module/README.md:41]: Do we have an issue to improve this error message?

**Review by millsp — APPROVED** [review]
Nice cleanup!

File: dataproxy/cloudflare-workers-module/wrangler.base.toml (hunk 1 above)
Comment (Jolg42) [reply dataproxy/cloudflare-workers-module/wrangler.base.toml]: Or actually, remove it?

File: dataproxy/cloudflare-workers-module/README.md (hunk 2 above)
Comment (Jolg42) [reply dataproxy/cloudflare-workers-module/README.md:41]: I don't think so

File: dataproxy/cloudflare-workers-module/wrangler.base.toml (hunk 1 above)
Comment (Jolg42) [reply dataproxy/cloudflare-workers-module/wrangler.base.toml]: Updated :check:

File: dataproxy/cloudflare-workers-module/README.md (hunk 2 above)
Comment (janpio) [reply dataproxy/cloudflare-workers-module/README.md:41]: Does it fail with the same error message on a real Cloudflare deployment?
(I see this is from your local system with wrangler)

**Pull request merged**